### PR TITLE
Add more descriptive READMEs

### PR DESCRIPTION
**Pull request by c00k133**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,78 @@
-# Compiling Protocol Buffers
+<div align="center">
```
Comment (wgslr) [inline README.md:1]: Fancy 😏

File: README.md [hunk 2]
```diff
@@ -1,18 +1,78 @@
...
+
+First build a production build (run from root folder):
+
+  yarn build:prod
+
+The execute the production build with:
+
+  yarn exec:prod
+
+The app should now be exposed at `laclhost:3001`.
```
Comment (wgslr) [inline README.md:74]:
```suggestion
The app should now be exposed at `localhost:3001`.
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,78 @@
...
+
+--------------------------------------------------------------------------------
+
+_If you want to setup a server configuration on your own, see_ `server_configuration`.
```
Comment (wgslr) [inline README.md:78]:
```suggestion
_If you want to setup a public server configuration on your own, see_ `server_configuration`.
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,78 @@
...
+  yarn bootstrap
+
+### Running Development Build
+
+To run a development setup, the easiest way is to run the following in the root folder:
+
+  yarn start:dev
+
+This will start the server and a client for you.
+The client is expose at `localhost:3000` by default.
```
Comment (wgslr) [inline README.md:60]:
```suggestion
The client is exposed at `localhost:3000` by default.
```

**Review by wgslr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (c00k133) [reply README.md:1]: Guess where I got it from: https://github.com/wgslr/encrypt-to-me-rage/blob/master/README.md

Comment (wgslr) [reply README.md:1]: Hahaha. And I didn't even write that:P

**Review by wgslr — APPROVED** [review]

**Pull request merged**